examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+# Deferred Processing and Transactions
+
+## Introduction
+[Deferred processing](https://docs.pachyderm.com/latest/how-tos/deferred_processing/) 
+is a Pachyderm technique for controlling when data gets processed.
+Transactions are Pachyderm feature for assuring data from different repos get processed together.
+
+This is an example which uses a simple DAG, 
+based on our opencv example, 
+to illustrate using deferred processing along with transactions.
+
+
+
+## Pipelines
+
+![Example DAG](./example_dag.png)
```

Comment:
Add some text before the image. For example:

The following diagram demonstrates the DAG that is used in this example.